pkg/cmd/tknpac/bootstrap/bootstrap.go

```diff
@@ -222,30 +227,31 @@ func GithubApp(run *params.Run, ioStreams *cli.IOStreams) *cobra.Command {
 }
 
 func DetectPacInstallation(ctx context.Context, wantedNS string, run *params.Run) (string, error) {
-	// detect which namespace pac is installed in
-	// verify first if the targetNamespace actually exists
-	if wantedNS != "" {
-		installed, err := checkNS(ctx, run, wantedNS)
-		if err != nil {
-			return "", err
-		}
-		if !installed {
-			return "", fmt.Errorf("PAC is not installed in namespace: %s", wantedNS)
-		}
-		return wantedNS, nil
-		// if openshift pipelines ns is installed try it from there
+	_, err := run.Clients.Extension.ApiextensionsV1().
+		CustomResourceDefinitions().Get(ctx, repositoryCRD, metav1.GetOptions{})
```

Comment:
this didn't work with the repository client? need to import apiextensions CRD for that?